.travis.yml

```diff
@@ -27,11 +29,11 @@ jobs:
 
   # Testing on Node.js
   - env: NODE_VERSION=stable
-    dart_task: {test: -t node}
+    dart_task: {test: -x snapshot}
```

Comment:
Why run non-Node tests here and below?

Reply:
I forgot about the `Patch` tests and was thinking that every test was either tagged `node` or `snapshot`. Making the snapshot optional eliminated this need for this tag, so I changed it back.